src/main/java/com/google/sps/servlets/NewRecipeServlet.java

```diff
@@ -30,37 +31,31 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String description = request.getParameter("description");
     String ingredientsResponse = request.getParameter("ingredients");
     String stepsResponse = request.getParameter("steps");
-    // privacy will be used once we give users the option to make their recipes public
-    // String privacy = request.getParameter("privacy");
+    String privacy = request.getParameter("privacy");
```

Comment:
consider using boolean isPublic for URL parameter, smaller potential for usage mistakes.

Reply:
if it was a boolean then I would but I think renaming it that would cause more confusion